life_expectancy/cleaning.py

```diff
@@ -26,16 +30,32 @@ def clean_data(region_name: str):  # pylint: disable=C0116
     data = data[data['value'] != ' ']
     data = data[data['region'] == region_name]
     data['value'] = data['value'].astype(float)
+    return data
+
+
+def save_data(data) -> csv:  # pylint: disable=C0116
```

Comment:
In this case, `csv` is a python module (just like, for example `life_expectancy.cleaning` is a python module) so that is not the correct type.

In your case, you can see there's no `return` command, so the return type should be `None` as you are not actually returning anything (actually, you are creating a new file, but there is no way to explain that with type hints... (but you can explain that inside a docstring!))